test/functional/test_framework/mininode.py

```diff
@@ -1749,7 +1749,7 @@ def got_data(self):
     def send_message(self, message, pushbuf=False):
         if self.state != "connected" and not pushbuf:
             raise IOError('Not connected, no pushbuf')
-        logger.debug("Send message to %s:%d: %s" % (self.dstaddr, self.dstport, repr(message)[:500]))
+        self._log_message(True, message)
```

Comment:
nit: Instead of a unnamed boolean, you could pass in the string "Sent" or "Received" directly?